backend_application/app/crawler/pagerank.py

```diff
@@ -69,21 +78,26 @@ def get_probabilyties(session, new_x):
     :type new_x: np.array.
     :return: matrix.
     """
-    r = get_rows(session)
+    r = rows
```

Comment:
I think you should omit 'r' variable, and keep the rows variable, instead of the r. Also you have to correct/remove false docstring